internal/v1/api.yaml

```diff
@@ -1684,6 +1686,21 @@ components:
           type: boolean
           description: Enables the system FIPS mode
           default: false
+    Installer:
+      type: object
+      additionalProperties: false
+      description: Anaconda installer configuration
+      properties:
+        unattended:
+          type: boolean
+          description: |
+            Create a kickstart file for a fully automated installation
```

Comment:
Like on the installed system in `/root`?

Reply:
On the ISO.  It's under the `Installer` customizations which are described as `Anaconda installer configuration` and only apply to ISO builds.